room/compiler/src/test/kotlin/androidx/room/processor/QueryMethodProcessorTest.kt

```diff
@@ -734,6 +733,30 @@ class QueryMethodProcessorTest(val enableVerification: Boolean) {
         }.compilesWithoutError()
     }
 
+    @Test
+    fun skipVerificationPojo() {
+        singleQueryMethod<ReadQueryMethod>(
+            """
+                @SkipQueryVerification
+                @Query("SELECT bookId, uid  FROM User")
+                abstract NotAnEntity getPojo();
+                """
+        ) { parsedQuery, _ ->
+            assertThat(parsedQuery.name, `is`("getPojo"))
+            assertThat(parsedQuery.parameters.size, `is`(0))
+            assertThat(
+                parsedQuery.returnType.typeName,
+                `is`(COMMON.NOT_AN_ENTITY_TYPE_NAME as TypeName)
+            )
+            val adapter = parsedQuery.queryResultBinder.adapter
+            assertThat(checkNotNull(adapter))
+            assertThat(adapter::class, `is`(PojoRowAdapter::class))
+            val rowAdapter = adapter.rowAdapter
+            assertThat(checkNotNull(rowAdapter))
+            assertThat(rowAdapter::class, `is`(PojoRowAdapter::class))
+        }
```

Comment:
You need to invoke `.compilesWithoutError()`, otherwise it does not really run anything. Once you do that you might notice that the first assertion, `assertThat(adapter::class, `is`(PojoRowAdapter::class))`, is not quite right and likely needs to be asserting that the 'result adapter' is a `SingleEntityQueryResultAdapter`.